index.js

```diff
@@ -358,7 +358,9 @@ module.exports = (command, args, options) => {
 	spawned.cancel = function () {
 		if (!spawned.killed) {
 			canceled = true;
-			spawned.kill();
+			if (!spawned.kill()) {
+				canceled = false;
+			}
 		}
```

Comment:
If you call twice `cancel()`, the second time `canceled` will be set back to `false`.

This could be fixed by doing instead:

```js
spawned.cancel = function() {
  if (spawned.killed) { return; }

  if (spawned.kill()) {
    canceled = true;
  }
};
```